CMSgov/easi-app
# [EASI-3245] Manage LCID selection page

# EASI-3245

[Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=429%3A111458&mode=design&t=4a1PIuDSqRfL3tSA-1)

## Changes and Description

- Created Manage LCID selection page (src/views/GovernanceReviewTeam/Actions/ManageLcid/index.tsx)
  - Conditional logic for rendering title, description, and options (see [Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=432%3A118090&mode=design&t=4a1PIuDSqRfL3tSA-1))
  - Summary box renders only available actions
- Routes and Initial form layout for manage LCID forms
  - Retire (/governance-review-team/:sytemId/manage-lcid/retire)
  - Update (/governance-review-team/:sytemId/manage-lcid/update)
  - Expire (/governance-review-team/:sytemId/manage-lcid/expire)



## How to test this change

1. Manually set LCID status for testing by replacing code on [line 71 of src/views/GovernanceReviewTeam/Actions/index.tsx](https://github.com/CMSgov/easi-app/blob/7bb8d68550646a935f8ccbd807ba334333c7e5ea/src/views/GovernanceReviewTeam/Actions/index.tsx#L71C8-L71C8) with code below:
```jsx
const { state } = systemIntake;

const decisionState = SystemIntakeDecisionState.LCID_ISSUED;

// Replace ISSUED with RETIRED or EXPIRED to test other lcid statuses
const lcidStatus = SystemIntakeLCIDStatus.ISSUED;
```
2. Enable `itGovV2Enabled` flag
3. Go to /governance-review-team/:sytemId/actions
4. Select Manage a Life Cycle ID
5. Ensure selection page and forms work as expected

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
